.github/workflows/ci.yml

```
@@ -22,5 +22,5 @@ jobs:
     - name: Resolve dependencies
       run: xcodebuild -resolvePackageDependencies
     - name: Build and Run tests
-      run: xcodebuild -scheme BookPlayer test -testPlan Unit\ Tests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.0.1'
+      run: xcodebuild -scheme BookPlayer test -testPlan Unit\ Tests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.0'
```

```suggestion
      run: xcodebuild -scheme BookPlayer test -testPlan Unit\ Tests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.2'
```
https://github.com/TortugaPower/BookPlayer/actions/runs/7262525331/job/19785852622#step:7:48

This change breaks the build. there is no OS=17.0

---

yeah I have to circle back to this, in the original build the tests never kicked off for some reason [[job]](https://github.com/TortugaPower/BookPlayer/actions/runs/7249398846/job/19767546775), so I was testing with [macos13-arm64](https://github.com/actions/runner-images/blob/main/images/macos/macos-13-arm64-Readme.md) but the job never started for that, so I'm guessing it's only for some paid plan. I'll test out with 17.2 and hopefully the tests start without problems

---

yup that did it, thanks @SimonHaas